File: src/main/webapp/resources/js/Document.js
class Document {
	constructor(
		id,
		name,
		description,
		uploadDate,
		documentFile,
		requestId
	) {
		this._id = id;
		this._name = name;
		this._description = description;
		this._uploadDate = uploadDate;
		this.documentFile = documentFile
		this._requestId = requestId;
	}
	
	get id() {
		return this._id;
	}
	
	set id(id) {
		this._id = id;
	}
	
	get name() {
		return this._name;
	}
	
	set name(name) {
		this._name = name;
	}
	
	get description() {
		return this._description;
	}
	
	set description(description) {
		this._description = description;
	}
	
	get uploadDate() {
		return this._uploadDate;
	}
	
	set uploadDate(uploadDate) {
		this._uploadDate = uploadDate;
	}
	
	get documentFile() {
		return this._documentFile;
	}
	
	set documentFile(documentFile) {
		this._documentFile = documentFile;
	}
	
	get requestId() {
		return this._requestId;
	}
	
	set requestId(requestId) {
		this._requestId = requestId;
	}
}

File: src/main/java/pe/com/emilima/serviciodocumental/dto/Document.java
package pe.com.emilima.serviciodocumental.dto;

import java.util.Date;

public class Document {
	private int id;
	private String name;
	private String description;
	private Date uploadDate;
	private int fileId;

	public Document() {

	}

	public Document(int id, String name, String description, Date uploadDate, int fileId) {
		this.id = id;
		this.name = name;
		this.description = description;
		this.uploadDate = uploadDate;
		this.fileId = fileId;
	}

	public int getId() {
		return id;
	}

	public void setId(int id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getDescription() {
		return description;
	}

	public void setDescription(String description) {
		this.description = description;
	}

	public Date getUploadDate() {
		return uploadDate;
	}

	public void setUploadDate(Date uploadDate) {
		this.uploadDate = uploadDate;
	}

	public int getFileId() {
		return fileId;
	}

	public void setFileId(int fileId) {
		this.fileId = fileId;
	}
}
